<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eCatName"/>
     </item>
     <item>
      <widget class="QPushButton" name="bCatSearch">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bCatNew">
       <property name="text">
        <string>Nuevo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bCatDelete">
       <property name="text">
        <string>Borrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="v_categories">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eCatName</tabstop>
  <tabstop>bCatSearch</tabstop>
  <tabstop>bCatNew</tabstop>
  <tabstop>bCatDelete</tabstop>
  <tabstop>v_categories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
